convert_state.py

```diff
@@ -1,60 +1,66 @@
+"""
+CLI program convert state names to abbreviations and vice versa
+"""
 
+import json
 
 
-import json  # import json for json processing
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
-
-    print(data)
-    
-    # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
-
+    """Allow user to get conversions from dicts"""
+    state_names, state_abbreviations = get_names_and_abbreviations()
 
-    #loop
     while True:
         print('1. Convert state to abbreviation')
-        print("2. Convery abreviasion to state")
-        print(' 3. quit')
-        choice =   input("Enter choice : ")
-
-        if choice=="1":
-
-            convertStateToAbbreviation(stateAbbr2)
-        elif  choice == "2":
-            convert_abbreviation_to_state(stateAbbr)
-        
-        
-        elif choice==  "3":
+        print('2. Convery abreviasion to state')
+        print('3. Quit')
+        choice =   input('Enter choice: ')
+        if choice == '1':
+            convert_state_to_abbreviation(state_names)
+        elif  choice == '2':
+            convert_abbreviation_to_state(state_abbreviations)
+        elif choice == '3':
             break
         else:
-            print('try again')
-def convertStateToAbbreviation(dictionary ):
-    userInput   = input("Enter state name").  capitalize()
-    result = dictionary.get(userInput  )
-    if result == None:
-        print(' state not found')
-    else:
-        print ( "The abbreviation for " +  userInput+" is " + result)
-def convert_abbreviation_to_state(dictionary):
-    userInput   = input("Enter abbrviation name").upper()
-    result = dictionary.get(userInput)
-    if result == None:
-        print(' abreviation not found')
-    else:
-        print("the state with the abbreviation   " + userInput + " is " + result)
+            print('Try again')
+
 
+def get_names_and_abbreviations():
+    """
+    Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states
+    """
+    with open('data.json','r') as file:
+        abbreviation_json = file.read()
+        state_abbreviations = json.loads(abbreviation_json)
+    print(state_abbreviations)
+    
+    # Name is key in name dict, value in abbreviation dict. 
+    # Abbreviation is key in abbreviation dict, value in state dict.
```

Comment:
Using names like states_names_to_abbreviations would mean you can avoid comments like this